src/binary/int.rs

```diff
@@ -197,7 +197,7 @@ impl From<DecodedInt> for Coefficient {
         } = int;
         use types::Sign::{Negative, Positive};
         let sign = if is_negative { Negative } else { Positive };
-        Coefficient::new(sign, value)
+        Coefficient::new(sign, value.unsigned_abs())
```

Comment:
🗺️ All signed int primitives have an `unsigned_abs()` method that returns their absolute value as an unsigned integer of the closest-sized type (`i8`->`u8`, `i16`->`u16`, etc). Now the `Int` type has an `unsigned_abs()` method that returns a `UInt`.